src/rlgl.h

```diff
@@ -789,10 +789,16 @@ RLAPI void rlLoadDrawQuad(void);     // Load and draw a quad
 #endif
 
 #if defined(GRAPHICS_API_OPENGL_ES2)
+
+    #if defined(PLATFORM_DESKTOP)
+    #define GLAD_GLES2_IMPLEMENTATION
+    #include "external/gles2.h"
```

Comment:
This file should probably be named `glad_gles2.h` instead for clarity and to avoid confusion with gl specific headers.

Reply:
I renamed it.